docs/enhancements/e2e-command-line.md

```diff
@@ -0,0 +1,78 @@
+# Sub Command for E2E Results Viewing
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Objectives](#objectives)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+  * [User Stories](#user-stories)
+* [Unresolved Questions](#unresolved-questions)
+
+## Summary
+
+End to end (e2e) tests in the Kubernetes ecosystem are built on top of ginkgo
+and output junit XML. XML is not designed for human readability and thus it is
+very hard to grok. Users can use scanner but if they have results there is no
+way for a user to upload those results to scanner to get the nice output.
+
+## Objectives
+
+Make the sonobuoy e2e output be more useful to users.
+
+### Goals
+
+- Output a list of e2e tests (filterable by passed/failed).
+- Generate YAML that can rerun failed tests.
+
+### Non-Goals
+
+- Replace scanner.heptio.com
+- Updating scanner features
+- Re-implement gen
+- Make a CLI per plugin. This is a special case that generates a lot of
+  questions and we think this will improve the user experience by a lot.
+
+## Proposal
+
+Add a subcommand to `sonobuoy` that has arguments/subcommands that satisfies the
+goals above:
+
+`sonobuoy e2e` will:
+
+1. download the data from the blocking sonobuoy run on the cluster
+2. Parse the results file (or error if it cannot find e2e results)
+3. Display the list of failed tests followed by thee list of passed tests.
+   Ignore skipped tests.
+
+`sonobuoy e2e <path/to/results_archive>` will do steps 2 and 3 from above.
+
+`sonobuoy e2e --show=passed` will only display passed tests `sonobuoy e2e
+--show=failed` will only show failed tests
```

Comment:
where do we actually generate the yaml?

Reply:
I need to think more about the UX there...

Reply:
@liztio do you mean for the rerun command? It would be in the operator form the e2e subcommand I'm guessing unless as i'm writing it it fits better in the cmd.

@timothysc Can you be a bit more wordy about what you're thinking? My thought behind --show was that I hate the word "filter" because does it mean filter out or filter for? I think show 
is very explicit for this, but am not at all wed to it. Happy to take suggestions and brainstorm if you can tell me what you think the problem is/what you want to optimize for.

Reply:
It's just a flag default question.  Most folks who run this will want to see a summary of what failed, or if it all passed.  

So I think defaulting `--show=failed` makes sense, with an options plugin.d YAML output.